docs/explanation/launchpad.rst

```diff
@@ -0,0 +1,294 @@
+
+Launchpad PPAs allow you to build installable :term:`Ubuntu` :term:`Packages <Package>` for 
+multiple :term:`Architectures <Architecture>` and to host them in your own software
+:term:`Repository`. 
+
+Using a PPA is straightforward; you don't need the approval of anyone, therefore users 
+will have to enable it manually. See :ref:`InstallPackagesFromPPA`.
+
+This is pretty useful when you want to test a change or show others that a change
+builds successfull or is installable. Some people have the special permission to trigger
+the :term:`autopkgtests <autopkgtest>` for :term:`Packages <Package>` in a PPA.
+
+.. tip::
+
+    You can ask in the :term:`IRC` channel ``#ubuntu-devel`` if someone can trigger 
+    :term:`autopkgtests <autopkgtest>` in your PPA if you don't have the permission.
+
+git-based workflow for the development of Ubuntu Source Packages
+----------------------------------------------------------------
+
+Launchpad hosts a :doc:`/reference/git-ubuntu` importer service that maintains
+a view of the entire packaging version history of :term:`Ubuntu` 
+:term:`Source Packages <Source Package>` using :term:`git` :term:`Repositories <Repository>`
+with a common branching and tagging scheme. The :doc:`/reference/git-ubuntu` :term:`CLI`
+provides tooling and automation that understands these :term:`Repositories <Repository>`
+to make the development of :term:`Ubuntu` itself easier.
+
+You can see the web-view of these repositories when you click on the "Code" tab of any
+:term:`Source Package` on Launchpad, for example: https://code.launchpad.net/ubuntu/+source/hello
+
+.. image:: ../images/explanation/launchpad/GitUbuntuRepositoryOfTheHelloPackage.png
+   :align: center
+   :width: 35 em
+   :alt: Screenshot of the Launchpad Code page for the hello source package with an arrow pointing to the Code tab. 
+
+Comment Markup
+--------------
+
+Unfortunately Launchpad only regognizes a very limited set of markup patterns when you write comments. 
+
+.. note::
+
+    
+    Support for a wider range of markup patterns is a very comman and old request/wish; 
+    take for example LP: `#391780 <https://bugs.launchpad.net/launchpad/+bug/391780>`_.
+    
+    You can "upvote" (mark yourself as effected) or leave a comment to this bug report to
+    show your support for the feature request.
+
+    **Reminder:** Please stay civil! The Launchpad team has only limited resources.
+
+
+The full range of pattern is documented `here <https://help.launchpad.net/Comments>`_. 
+The most important patterns are:
+
+URIs
+~~~~
+
+Launchpad can recognize ``http``, ``https``, ``ftp``, ``sftp``, ``mailto``,
+``news``, ``irc`` and ``jabber`` :term:`URI's <URI>`.
+
+.. note::
+
+    ``tel``, ``urn``, ``telnet``, ``ldap`` :term:`URI's <URI>`, relative 
+    :term:`URL's <URL>` like ``example.com`` and E-Mail addresses like 
+    ``test@example.com`` are **NOT** recognized. 
+
+
+Referencing Launchpad Bugs
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Synopsis
+^^^^^^^^
+
```

Comment:
I got a little confused here, I'm not sure synopsis is very descriptive as a title. It might help to have a little sentence between the title and the code block to explain it a bit?

Reply:
> I got a little confused here, I'm not sure synopsis is very descriptive as a title. It might help to have a little sentence between the title and the code block to explain it a bit?

I thought this is an established concept; like in this man page: https://manpages.ubuntu.com/manpages/lunar/en/man1/bash.1.html

But I agree that this can be confusing for someone not aware of this :/ There will be more sections like this where the synopsis will just show the syntax e.g. of a command. Do you have a general recommendation what to write there?

Reply:
> I'm also a bit confused by the table and what its purpose is - might be worth just clarifying that with a short introduction as well

Good Point! The input is what you type in a launchpad text field and the output shows how it will be rendered on Launchpad. I add a short description to explain this.

Reply:
> > I got a little confused here, I'm not sure synopsis is very descriptive as a title. It might help to have a little sentence between the title and the code block to explain it a bit?
> 
> I thought this is an established concept; like in this man page: https://manpages.ubuntu.com/manpages/lunar/en/man1/bash.1.html
> 
> But I agree that this can be confusing for someone not aware of this :/ There will be more sections like this where the synopsis will just show the syntax e.g. of a command. Do you have a general recommendation what to write there?

If it's the expected terminology then stick with it, since the typical user of the packaging guide can be expected to know it. Let's chalk it up to "a thing I just needed to learn" :)

Reply:
I changed "Synopsis" to "Pattern" and moved the entire Launchpad comment section into a reference article as the section is barely explanatory content.

Resolved with 3d3ffc9a8b2db303ff33bbc35872d7d980f8ca3e